files/static/style.css

```diff
@@ -0,0 +1,234 @@
+html, body {
+  width: 100%;
+}
+
+body {
+  margin: 0;
+  background-color: #f5f5f5;
+  font-family: Helvetica, "Roboto", Tahoma, sans-serif;
```

Comment:
Did you mean to have Roboto first? It sounds a bit off to have different fonts in the body vs headers.